### DISCUSSION TITLE
issue with RVC

### DISCUSSION THREAD

**Discussion by staphamou79** [erew123/alltalk_tts#462] [General]
I am a total noob at this. I went to Global settings >RVC settings> enable RVC> then click on update RVC Settings. The msg RVC setting update successful. However, "default character voice model "and " default Narrator voice model"are both showing disabled, and there is no option in the drop down. I also went to file explorer and navigate to models/rvc_voices and it is empty. I was under the impression that once you enable RVC, the system will automatically download the necessary models. What am I doing? What did I miss? Thank you a bunch for your help.  PS: I am trying this for my wife, her mom passed away and she misses her voice, I manage to salvage some recordings and trying to offer some healing, they had some unfinished business. So if you can , please help! { It is my Christmas gift to her }
![image](https://github.com/user-attachments/assets/2c7fd934-0212-4ffd-b332-5ec771a1ee0f)
![image](https://github.com/user-attachments/assets/9aa92d59-cd8e-4ff6-93f8-133cfb6bbc69)

**Comment by erew123** [comment]
Hi @staphamou79 Enabling RVC will download the RVC base files. The base model files act as a foundation for RVC providing the core functionality. RVC Voice files, on the other hand, are specific to individual voices, and they need to be downloaded separately and placed in the rvc_voices folder for RVC to use them. 

Disabled will also always be in the list, as this can be used to bypass RVC processing on TTS generation, as sometimes you may want to use RVC and sometimes not use RVC.

Please see here on the WIKI for details of where you can download voice files.

https://github.com/erew123/alltalk_tts/wiki/RVC-(Retrieval%E2%80%90based-Voice-Conversion)#voice-model-files

Thanks
